lnd_test.go

```diff
@@ -306,28 +308,57 @@ func testChannelForceClosure(net *networkHarness, t *harnessTest) {
 	// the channel. This will also assert that the commitment transaction
 	// was immediately broadcast in order to fulfill the force closure
 	// request.
-	closeUpdate, err := net.CloseChannel(ctxb, net.Alice, chanPoint, true)
+	_, closingTxID, err := net.CloseChannel(ctxb, net.Alice, chanPoint, true)
 	if err != nil {
 		t.Fatalf("unable to execute force channel closure: %v", err)
 	}
 
+	if err := net.RestartNode(net.Alice, nil); err != nil {
```

Comment:
Minor nit: can you add a comment here explaining that this restart is to test that the nursery saves the "to be swept" outputs to disk in order to survive restarts? Thanks!